src/sentry/migrations/0471_backfill_organization_member_is_active.py

```diff
@@ -0,0 +1,40 @@
+# Generated by Django 2.2.28 on 2023-04-25 20:25
+
+from django.db import migrations
+
+from sentry.new_migrations.migrations import CheckedMigration
+from sentry.utils.query import RangeQuerySetWrapperWithProgressBar
+
+
+def backfill_om_is_active(apps, schema_editor):
+    OrganizationMember = apps.get_model("sentry", "OrganizationMember")
+    for om in RangeQuerySetWrapperWithProgressBar(OrganizationMember.objects.all()):
+        # Default is true, so update only on False.
+        if not om.user.is_active:
+            om.update(user_is_active=False)
```

Comment:
`.update` doesn't work in migrations, you'll need to set the field and use `.save(update_fields=["user_is_active"])`

Reply:
Thank you dude!!